src/ChannelServer/Network/ChannelPacketHandler.cs

```diff
@@ -650,6 +650,35 @@ public void CZ_HEAD_ROTATE(ChannelConnection conn, Packet packet)
 		}
 
 		/// <summary>
+		/// Sent when the character requests to use an item.
+		/// </summary>
+		/// <param name="conn"></param>
+		/// <param name="packet"></param>
+		[PacketHandler(Op.CZ_ITEM_USE)]
+		public void CZ_ITEM_USE(ChannelConnection conn, Packet packet)
+		{
+			var worldId = packet.GetLong();
+			var handle = packet.GetInt();
+
+			// Get item.
+			var item = conn.SelectedCharacter.Inventory.GetItem(worldId);
+			if (item == null)
+			{
+				Log.Warning("CZ_ITEM_USE: User '{0}' tried to use a non-existent item.", conn.Account.Name);
+				return;
+			}
+
+			// Do not allow use of locked items.
+			if (item.IsLocked)
+			{
+				Log.Warning("CZ_ITEM_USE: User '{0}' tried to use a locked item.", conn.Account.Name);
+				return;
+			}
+
+			// TODO: Implement use of item.
```

Comment:
I left out the rest of the implementation, but included a stub because in order to record client-side logic that we should also verify server side too.

Reply:
A server message would be good, to let the user know that the feature isn't actually working yet, and they don't wonder why *nothing* happened, not even an error or an unimplemented message in the channel.